### DISCUSSION TITLE
[Nuxt] useStorage not working when deployed but locally yes

### DISCUSSION THREAD

**Discussion by impons** [vercel/community#1396] [Help]
### Summary

Hello! I'm using `useStorage`  with this configuration:
```
const storageBase = `${process.cwd()}/.data/db` ;
...
nitro: {
    storage: {
      'db': {
        driver: 'fs',
        base: storageBase,
      },
    },
  },
```

In my local works correctly but when deployed to vercel and the endpoint tries to use the storage gives the next error:
```
ENOENT: no such file or directory, open '/vercel/path0/.data/db/invitados'
```

Any ideas what could be happening?
Thanks.

**Comment by marc-tech** [comment]
+1 with the last version of nuxt/i18n 8.0.0-beta.11
https://github.com/nuxt-modules/i18n/issues/2004
